application-arrow/src/commonMain/kotlin/com/fraktalio/fmodel/application/EphemeralViewArrowExtension.kt

```diff
@@ -0,0 +1,35 @@
+package com.fraktalio.fmodel.application
+
+import arrow.core.Either
+import arrow.core.raise.catch
+import arrow.core.raise.either
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.fold
+
+
+suspend fun <S, E, Q, MV> MV.handleWithEffect(query: Q): Either<Error, S>
```

Comment:
MV stands for MaterializedView. Should we consider EV for this case?

Reply:
Done, thanks!